### PR TITLE
Update train.json

### PR DESCRIPTION
**Pull request by kjonosm**
Update train networks in Germany

### CODE REVIEW

File: data/transit/route/train.json [hunk 1]
```diff
@@ -393,12 +393,16 @@
...
     },
     {
-      "displayName": "Dublin Commuter",
+      "displayName": "Commuter",
       "id": "dublincommuter-73b98a",
       "locationSet": {"include": ["ie"]},
       "tags": {
-        "network": "Dublin Commuter",
-        "operator": "Dublin Commuter",
+        "network": "Commuter",
```
Comment (bhousel) [inline data/transit/route/train.json:400]: If the `network=Dublin Commuter` is already in widespread use, we probably shouldn't encourage retagging it to `network=Commuter`.  But other than that, the rest of the PR looks good 👍

**Review by bhousel — CHANGES_REQUESTED** [review]
just a question about the Dublin Commuter

File: data/transit/route/train.json (hunk 1 above)
Comment (kjonosm) [reply data/transit/route/train.json:400]: OK. Unfortunately I don't know how to edit the pr to fix the issue...

Comment (bhousel) [reply data/transit/route/train.json:400]: > OK. Unfortunately I don't know how to edit the pr to fix the issue...

thats ok, I'll merge and put it back

**Review by bhousel — APPROVED** [review]

**Pull request merged**